src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.html

```diff
@@ -5,19 +5,12 @@
   [class.media-unavailable]="isMediaUnavailable()"
   sofieTooltip
   [sofieTooltipTemplate]="pieceTooltip"
-  (tooltipMetadataChange)="updatePositionInVideo($event)"
->
+  (tooltipMetadataChange)="updatePositionInVideo($event)">
   <div class="label" [style.margin-left.px]="labelOffsetInPixels">
-    <span #labelTextElement>{{ piece.name }}</span>
+    <span *ngIf="piece.isSpanned !== true" #labelTextElement>{{ piece.name }}</span>
```

Comment:
Could be written as `*ngIf="!piece.isSpanned`

Reply:
It could be, I just personally thought that it was more telling that I was expecting to check for the true statement :) After having looked at !piece.isSpanned for the whole period I was developing, I ended up choosing the semantic "if piece isSpanned is not true" instead of "if not piece isSpanned".

I could argue for both sides, but will leave it up to the team to decide about this :)